scripts/main/sidebar.js

```diff
@@ -73,6 +80,18 @@ sidebar.bind = function() {
 
 };
 
+sidebar.triggerSearch = function(search_string) {
+
+	// If public search is diabled -> do nothing
+	if (lychee.publicMode===true && !lychee.public_search) {
+		// Do not display an error -> just do nothing to not confuse the user
+		return;
+	}
+
+	// We're either logged in or public search is allowed
+	lychee.goto('search/' + encodeURIComponent(search_string));
```

Comment:
I think you need to reset `search.hash` to `null` before this call (the way `search.reset` does it).

I'm seeing a problem where clicking on a tag only works the first time. If I click on the same tag again, without any other searches in-between, nothing happens, and the reason is that `search.hash` was not reset in the meantime.

Reply:
should be resolved